### DISCUSSION TITLE
ASN Information in Aggregates

### DISCUSSION THREAD

**Discussion by digital-fortress** [netbox-community/netbox#14116] [Ideas]
I currently track ASN details in aggregates on NetBox using a custom field, as shown in the attached image. Is it possible to make this a standard field within the application?

Additionally, I would like to explore the feasibility of having the same ASN automatically populated for sites within the prefix.
![image](https://github.com/netbox-community/netbox/assets/13614357/14c30bd3-2dcc-434d-9cb1-3eaf2ec8d11e)

**Comment by alasdairmuckart** [comment]
I would really like to see a link between the ASN model and the Aggregate model.

It seems strange to me that they both exist but aren't connected at all.

It obviously can't be mandatory because not all Aggregates have an ASN, but all Aggregates under a public RIR _should_.
